walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -112,4 +99,10 @@
 
     return timeRangesSet;
   }
+
+  public List<TimeRange> addDummyEvents(List<TimeRange> timeRanges){
+    timeRanges.add(TimeRange.fromStartDuration(TimeRange.START_OF_DAY, 0));
+    timeRanges.add(TimeRange.fromStartDuration(TimeRange.END_OF_DAY+1, 0));
```

Comment:
nit: We usually put spaces between non-unary operators (`+`) and their operands (`TimeRange.END_OF_DAY` and `1`).

`TimeRange.END_OF_DAY + 1` rather than `TimeRange.END_OF_DAY+1`

Reply:
Added all these changes to PR #24